meta-mender-core/classes/mender-full.bbclass

```diff
@@ -12,7 +12,7 @@ MENDER_FEATURES_ENABLE_append = " \
 _MENDER_IMAGE_TYPE_DEFAULT ?= "mender-image-uefi"
 _MENDER_BOOTLOADER_DEFAULT ?= "mender-grub"
 
-_MENDER_GROWFS_DATA_DEFAULT ?= "${@'' if d.getVar('MENDER_EXTRA_PARTS') else 'mender-growfs-data'}"
+_MENDER_GROWFS_DATA_DEFAULT ?= "${@'' if d.getVar('MENDER_EXTRA_PARTS') else 'mender-client-growfs-data'}"
```

Comment:
You don't need the client to grow the filesystem, so I'd argue "mender" here as well.